src/app/ui/multiParamManager.js

```diff
@@ -207,7 +207,10 @@ class MultiParamManager {
     if (this.funABI.payable === true) {
       contractProperty.classList.add(css.payable)
       button.setAttribute('title', (title + ' - transact (payable)'))
-      this.contractActionsContainerSingle.querySelector('button').setAttribute('title', (title + ' - transact (payable)'))
+      let funButton = this.contractActionsContainerSingle.querySelector('button')
+      funButton.setAttribute('title', (title + ' - transact (payable)'))
+      funButton.classList.add('btn-danger')
```

Comment:
With this code the btn-danger is getting added to the css - but the btn-warning is still there - it should get removed or really it should get put on there in the 1st place.  The code works as is - but its a bit messy because both classes are there - and the last class which was added - btn-danger wins - because it was the last added.

Reply:
@LianaHus @ryestew it looks like that the comment was addressed